metricbeat/module/apache/status/data.go

```diff
@@ -15,6 +15,42 @@ var (
 
 	// This should match: "CPUSystem: .01"
 	matchNumber = regexp.MustCompile("(^[0-9a-zA-Z ]+):\\s+(\\d*\\.?\\d+)")
+
+	conversions = common.MapStr{
```

Comment:
this almost looks like a schema/conversion defintion. Next step: make conversions table loadable from file?